enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotAdmin/MetabotAdminSuggestedPrompts.unit.spec.tsx

```diff
@@ -76,6 +76,11 @@ describe("suggested prompts", () => {
   });
 
   it("should successfully render a list of prompts", async () => {
+    await setup();
+    await expectVisiblePrompts(defaultMetabotMockedPrompts.slice(0, 3));
+  });
+
+  it("should show loading state", async () => {
     setupMetabotPromptSuggestionsEndpoint(
```

Comment:
this diff makes it look more confusing than it is. new lines 84-94 were originally in the `should successfully render a list of prompts"` test and now they're in the `"should show loading state"` test.